src/3_PB/AnalisiDeiRequisiti/content.typ

```diff
@@ -90,55 +89,51 @@ Relativamente all'utilizzo della #glossary("dashboard"), viene definito un unico
+=== UC1.1: Visualizzazione #glossary("pannello") dati grezzi
 - *Attore Principale*: #glossary("amministratore pubblico")\;
-- *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
-- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente una mappa che mostra le posizioni dei sensori, visualizzati tramite icone, su di essa. Le icone dei sensori sono colorate in base al tipo di #glossary("sensore")  e riportano una label che ne esplicita il tipo, il nome e le coordinate geografiche; 
+- *Precondizioni*: l'#glossary("amministratore pubblico") ha selezionato la visualizzazione relativa al dominio dei dati grezzi;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente i dati grezzi;
 - *Scenario Principale*:
-  + L'#glossary("amministratore pubblico") accede alla piattaforma di visualizzazione; 
-  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary("dashboard") generale relativa ai sensori.
+  + L'#glossary("amministratore pubblico") accede alla piattaforma di visualizzazione;
+  + L'#glossary("amministratore pubblico") seleziona la visualizzazione dei dati grezzi.
+- *Specializzazioni*: [UC1.2];
 - *Estensioni*: [UC9].
 
 #figure(
-  image("assets/UML/UC1.1_Visualizzazione-posizione-sensori-su-mappa.png",width:100%),
-  caption: [UC1.1 Visualizzazione posizione sensori su mappa]
+  image("assets/UML/UC1.1_Visualizzazione-pannello-dati-grezzi.png",width:100%),
+  caption: [UC1.1 Visualizzazione #glossary("pannello") dati grezzi]
 )
 
-=== UC1.2: Visualizzazione tabella sensori
+=== UC1.2: Visualizzazione tabella dati grezzi
 - *Attore Principale*: #glossary("amministratore pubblico")\;
 - *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
-- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza il #glossary("pannello") relativo ai dati inerenti ai sensori, che riporta il nome del #glossary("sensore"), la percentuale di batteria, un valore booleano che esprime se il #glossary("sensore") è alimentato da batteria autonoma o meno e la data di ultima manutenzione del #glossary("sensore"), in forma tabellare; la tabella contenente i dati menzionati è indicizzata tramite il nome dei sensori;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza il #glossary("pannello") relativo ai dati grezzi, che riporta il timestamp della misurazione, il valore della misurazione, la latitudine e la longitudine del #glossary[sensore], in forma tabellare, per ogni tipo di #glossary[sensore]; la tabella contenente i dati menzionati è indicizzata tramite il nome dei sensori. Nel caso in cui il #glossary[sensore] invii altri dati contestuali alla misurazione principale, come nel caso dei sensori che monitorano le colonne di ricarica [UC17], la congestione stradale [UC20] o il vento [UC12], questi saranno a loro volta visualizzabili all'interno della tabella.
```

Comment:
io mi limiterei a togliere l'ultimo frase delle postcondizioni così è meno specifico e non abbiamo fastidi di nessuno tipo relativamente all'implementazione attuale. 

per quanto riguarda divederlo in più sottocasi d'uso io non credo sia necessario, cioè va bene così com'è.

comunque ti invito ad aprire quella dashboard nel branch allerte per avere modo di vedere come è attualemente cosi da poter descrivere quello.